src/GitVersionExe/GitVersionExe.csproj

```diff
@@ -6,7 +6,7 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>GitVersion</RootNamespace>
     <AssemblyName>GitVersion</AssemblyName>
-    <TargetFrameworks>net40;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(PackAsTool)' != 'true'">net40;netcoreapp2.0</TargetFrameworks>
```

Comment:
Very minor thing, I understand why you have added this condition, shame it is necessary though, from a purity perspective I would find it nicer if this project file was oblivious to global tool builds, and all global tool build stuff was handled in `GitVersion.Tool.csproj`. However one condition like this is no biggie really. It makes me wonder though if it would work if you removed this condition , and then in GitVersion.Tool.csproj, after the `GitVersion.Tool.proj` file is imported, redeclare the TargetFrameworks property there - thinking it would override the imported value from here..
However this is not a blocker for merging this.

Reply:
I have changed this by importing the project first, and then overriding with the netcoreapp2.1 for the global tool